src/main/java/core/basesyntax/suppliers/figuresuppliers/RandomFigureSupplier.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.suppliers.figuresuppliers;
+
+import core.basesyntax.enums.Shape;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import core.basesyntax.suppliers.colorsuppliers.RandomColorSupplier;
+import java.util.Random;
+
+public class RandomFigureSupplier {
+    private final int maxSideLength;
+    private final int minSideLength;
+    private final RandomColorSupplier randomColorSupplier = new RandomColorSupplier();
+    private final Random random = new Random();
+
+    public RandomFigureSupplier(int minSideLength, int maxSideLength) {
+        this.minSideLength = minSideLength;
+        this.maxSideLength = maxSideLength;
+    }
+
+    public Figure getRandomFigure(Shape figureType) {
+        int[] parameters = generateParameters();
+        Figure figure;
+        switch (figureType) {
+            case CIRCLE: figure = new Circle(getRandomColor(), parameters[0]);
```

Comment:
Move it to the next line. Also, let's use variables.

E.g. `int radius = parameters[0]`.